app/models/gobierto_data/dataset.rb

```diff
@@ -10,12 +10,15 @@ class Dataset < ApplicationRecord
     include GobiertoData::Favoriteable
     include GobiertoAttachments::Attachable
     include GobiertoCommon::Collectionable
+    include GobiertoCommon::HasCustomFieldRecords
 
     belongs_to :site
     has_many :queries, dependent: :destroy, class_name: "GobiertoData::Query"
     has_many :visualizations, dependent: :destroy, class_name: "GobiertoData::Visualization"
 
     scope :sorted, -> { order(data_updated_at: :desc) }
+    scope :visibles, -> { where(visibility_level: "active") }
+    scope :by_site, ->(site_id) { where(site_id: site_id) }
```

Comment:
not necessary, see my comment below

Reply:
question:  not necessary :by_site +100 because I should use site.datasets

What about visible? ... I thing is useful for me at least at hiding draft datasets.

thx

Reply:
that scope is not necessary. The enum in the model gives you automatic scopes. So the query should be `current_site.datasets.active` ;)